indico/modules/events/management/forms.py

```diff
@@ -270,6 +268,20 @@ def _init_visibility(self, event):
             self.visibility.warning = _("This event's visibility is currently limited by that of '{}'.").format(
                 real_horizon.title)
 
+    def validate_permissions(self, field):
+        event = self.event
+        for idx, value in enumerate(field.data):
+            principal_fossil = value[0]
+            permissions = value[1]
+            principal = principal_from_fossil(principal_fossil, allow_emails=True, allow_networks=True, event=event)
+            if isinstance(principal, IPNetworkGroup):
+                if set(permissions) - {READ_ACCESS_PERMISSION}:
```

Comment:
if isinstance(principal, IPNetworkGroup) and set(permissions) - {READ_ACCESS_PERMISSION}: